project/redis-master.sls

```diff
@@ -0,0 +1,17 @@
+# Install Redis, listening on all interfaces.
+# This does not open the firewall for Redis, so do that elsewhere
```

Comment:
Maybe. Feels like just grabbing things out of projects is how this repo got to be in such a mess in the first place. I would say :-1: to opening on all interfaces without configuring the firewall and/or setting a password. See http://antirez.com/news/96

> The Redis security model is: “it’s totally insecure to let untrusted clients access the system, please protect it from the outside world yourself”